sources/models/src/modeled_types/oci_defaults.rs

```diff
@@ -0,0 +1,143 @@
+use scalar_derive::Scalar;
+use serde::{Deserialize, Serialize};
+
+/// OciDefaultsCapability specifies which process capabilities are
+/// allowed to be modified in Bottlerocket. The Linux process
+/// capabilities are defined in the Linux manpages (see the man7.org link below).
+/// Default values can be found in: sources/models/shared-defaults/oci-capabilities.toml
+/// OCI spec: https://github.com/opencontainers/runtime-spec/blob/c4ee7d12c742ffe806cd9350b6af3b4b19faed6f/config.md#linux-process
+/// Linux kernel capabilities: https://man7.org/linux/man-pages/man7/capabilities.7.html
+///
+/// Default capabilities are specified in the following file:
+/// * sources/models/shared-defaults/oci-capabilities.toml
```

Comment:
I'd probably clean up all of these links and just say:
```suggestion
/// OciDefaultsCapability specifies which process capabilities are
/// allowed to be set in the default OCI spec.
```

`sources/models/shared-defaults/oci-capabilities.toml` is extremely likely to be renamed during a refactor, and this comment probably won't stay correct. The OCI spec link seems a little out-of-place here, and the man page is pretty easy to find by anyone who knows they're working with Linux capabilities.

Reply:
Agreed - wouldn't want stale references in comments. Accepted suggestion